CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Minikube Release Notes
 
+## Version 0.16.0 - 2/2/2017
+* Updated minikube ISO to v1.0.6
```

Comment:
was thinking that release notes could link both minikube changelog and iso changelog

how do we link in docs?
to `master` filepath or `0.16` tag?